### ISSUE TITLE
Separate out Svg module from Html

### ISSUE THREAD

**Issue by Zaid-Ajaj** [Zaid-Ajaj/Feliz#251]
As described in [this comment](https://github.com/Zaid-Ajaj/Feliz/pull/249#issuecomment-673346806) to solve the ambiguities between Html attributes, Svg attributes and CSS properties. The idea is properly support SVG attributes alongside the CSS equivalent:
```fs
let getGradientDefinition (gradientId: string) (data: Point list) =
    let offset = getOffset data
    Svg.defs [
        Svg.linearGradient [
            svg.id gradientId
            svg.x1 0; svg.x2 0
            svg.y1 0; svg.y2 1
            svg.children [
                Svg.stop [
                    svg.offset offset
                    svg.stopColor "green"
                    svg.stopOpacity 1.0
                ]

                Svg.stop [
                    svg.offset offset
                    svg.stopColor "red"
                    svg.stopOpacity 1.0
                ]
            ]
        ]
    ]

svg.transform [
    svg.transform.translateX(10)
    svg.transform.translateY(20)
]
```
I suggest to implement it in a  non-breaking fashion: make current Svg attributes implemented in `Html` module `[<Obsolete>]` and re-implement them in `svg` and `Svg` modules. Wait until Fable 3 to make a proper upgrade to Feliz 2 with the breaking changes. 

How does that sound? @Shmew @zanaptak

**Comment by Zaid-Ajaj** [comment]
Would be awesome if you someone could take a stab at this one so that it will be added as part of the new stuff for Fable 3

**Comment by Shmew** [comment]
I wanted to do this but I'm kinda swamped right now with other projects. Depending on when Fable 3 actually is planned to release I *might* be able to swing back and grab it. Probably won't be for a couple weeks at least

**Comment by Zaid-Ajaj** [comment]
That's alright Cody, I know you are busy with really cool projects. I am keeping this issue open hoping more contributors would jump in if they want to since this one is quite straightforward :smile:

**Comment by Zaid-Ajaj** [comment]
Just FYI, this is now ⚠️ work in progress ⚠️ 

> since this one is quite straightforward

I think I will take this back :joy:

**Comment by Zaid-Ajaj** [comment]
Implemented as of v1.28 via #279
